guide/src/wasm-bindgen-test/continuous-integration.md

```diff
@@ -0,0 +1,63 @@
+# Setting Up Continuous Integration with `wasm-bindgen-test`
+
+This page contains example configurations for running `wasm-bindgen-test`-based
+tests in various CI services.
+
+Is your favorite CI service missing? [Send us a pull
+request!](https://github.com/rustwasm/wasm-bindgen)
+
+## Travis CI
+
+```yaml
+language: rust
+rust: nightly
+
+addons:
+  firefox: latest
+  chrome: stable
+
+install:
+  - rustup target add wasm32-unknown-unknown
+  - cargo install wasm-bindgen-cli
```

Comment:
I'd imagine that a bunch of people will just copy this so ideally this would be a fast-enough starting point. 

This can take minutes, so could it make sense to instead either:

option 1. Download a release binary and move it to either `~/.cargo/bin` or `/usr/local/bin`

option 2. Cache cargo deps (I see that the `wasm-bindgen` repo's travis seems to do this?)

Happy to do either if you think it makes sense!

Reply:
@chinedufn totally -- good call! I think option 1 is more straight forward, and we could also have a comment mentioning something like

> Download and unpack the `wasm-bindgen` CLI tools. Alternatively, use `wasm-pack` to manage `wasm-bindgen` binaries for you.

Thanks!

Reply:
Thanks for the guidance!

For posterity https://github.com/rustwasm/wasm-bindgen/pull/824